js/memory.js

```diff
@@ -0,0 +1,142 @@
+$(document).ready(function() {
+
+	(function() {
+		'use strict';
+
+		var cardAlreadyShowing = null;
+		var totalTries = 0;
+		var $boardEl = $('#board');
```

Comment:
Rather than making them nearly-global variables, where else could you put these?